Library/ViewModels/ApplePayTokenUseCase.swift

```diff
@@ -0,0 +1,160 @@
+import Foundation
+import KsApi
+import PassKit
+import ReactiveSwift
+
+public protocol ApplePayTokenUseCaseInputs {
+  func applePayButtonTapped()
+  func paymentAuthorizationDidAuthorizePayment(paymentData: (
+    displayName: String?,
+    network: String?,
+    transactionIdentifier: String
+  ))
+  func paymentAuthorizationViewControllerDidFinish()
+  func stripeTokenCreated(token: String?, error: Error?) -> PKPaymentAuthorizationStatus
+}
+
+public protocol ApplePayTokenUseCaseOutputs {
+  var goToApplePayPaymentAuthorization: Signal<PKPaymentRequest, Never> { get }
+  var applePayParams: Signal<ApplePayParams?, Never> { get }
+  var applePayAuthorizationStatus: Signal<PKPaymentAuthorizationStatus, Never> { get }
+}
+
+/**
+ A use case for ApplePay transactions in the regular (not post!) pledge flow.
```

Comment:
nit: I know that sometimes we use `live` instead of `post`. IMO `late` is a better naming convention, but either way It'd be a good idea to align on one and stick to it. You used `live` in your [other PR](https://github.com/kickstarter/ios-oss/pull/2255) so maybe we can go with that here.